### PR TITLE
feat(card-section): introducing web component versions

### PR DESCRIPTION
**Pull request by IgnacioBecerra**
### Related Ticket(s)

#3776 
#3778 

### Description

Introducing `<dds-card-section-simple>` and `<dds-card-section-images` for Web Components.

`Card Section Simple` contains the Default and With CTA variations:

Default:
Screen Shot 2020-11-12 at 2 52 27 PM

With CTA:
Screen Shot 2020-11-12 at 2 52 38 PM

`Card Section Images` only contains the Default variation:
Screen Shot 2020-11-12 at 2 52 51 PM

### Changelog

**New**

- `<dds-card-section-simple>` component
- `<dds-card-section-images>` component

### CODE REVIEW

File: packages/web-components/src/components/card-section-simple/card-section-simple.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      this.setAttribute('role', 'section');
+    }
+    super.connectedCallback();
+  }
+
+  render() {
+    return html`
+      ${super.render()}
+    `;
+  }
```
Comment (asudoh) [inline packages/web-components/src/components/card-section-simple/card-section-simple.ts:40]: If `super.render()` is the only thing, we don't have to override `render()`:

```suggestion
```

File: packages/web-components/src/components/card-section-simple/__stories__/card-section-simple.stories.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+          </div>
+        </div>
+      </div>
+    `,
+  ],
+  parameters: {
+    ...readme.parameters,
+    hasGrid: true,
+    hasVerticalSpacingInComponent: true,
+    gridLargeColumnClass: 'bx--col-lg-8',
```
Comment (asudoh) [inline packages/web-components/src/components/card-section-simple/__stories__/card-section-simple.stories.ts:111]: Seems that this is not used?

```suggestion
```

**Review by asudoh — COMMENTED** [review]
Good progress @IgnacioBecerra!

File: packages/web-components/src/components/card-section-simple/card-section-simple.ts (hunk 1 above)
Comment (IgnacioBecerra) [reply packages/web-components/src/components/card-section-simple/card-section-simple.ts]: Makes sense, thanks!

File: packages/web-components/src/components/card-section-simple/__stories__/card-section-simple.stories.ts (hunk 2 above)
Comment (IgnacioBecerra) [reply packages/web-components/src/components/card-section-simple/__stories__/card-section-simple.stories.ts]: Good catch!

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @IgnacioBecerra!

**Review by annawen1 — APPROVED** [review]
Nice! Thanks @IgnacioBecerra

**Pull request merged**